domain/src/main/assets/welcome.json

```diff
@@ -11,7 +11,29 @@
   "states": {
     "END": {
       "classifier_model_id": null,
+      "recorded_voiceovers": {
+        "voiceovers_mapping": {
+          "content": {
+            "en": {
+              "file_size_bytes": 0,
+              "needs_update": false,
+              "filename": "https://www.soundhelix.com/examples/mp3/SoundHelix-Song-1.mp3"
```

Comment:
I think we need to point to Oppia's GCS bucket since normally file names are just the flenames, not fully qualified. Otherwise the real implementation downloading the stream won't work correctly.

Reply:
Yeah I know. I was just putting full http links for testing purposes. I'm still waiting on Sean's response on how to get the GCS_RESOURCE_BUCKET_NAME.

Reply:
See my comment above--let's see if we can resolve this before Sean returns.